src/hooks/useRouter.ts

```diff
@@ -30,6 +30,7 @@ export const pages = {
     newExternalUserRequest: '/requests/users/new/external',
     newExternalFromMainUserRequest: '/requests/users/new/external-from-main',
     newExistingUserRequest: '/requests/users/new/existing',
+    newInternalUserRequestReadOnly: (requestId: string) => `/requests/users/internal/${requestId}`,
```

Comment:
just `internalUserRequest`, not `new`, because it's already created. And not `readOnly`, because it may depend on user access in the future